tests/css-values-5.js

```diff
@@ -104,23 +104,32 @@ export default {
 				tr: '#attr-notation',
 				dev: '#attr-notation',
 			},
-			properties: ['content', 'width', 'padding'],
-			tests: [
-				'attr(data-px)',
-				'attr(data-px, "12px")',
-				'attr(data-px px)',
-				'attr(data-px px, initial)',
-				'attr(data-string string)',
-				'attr(data-url url)',
-				'attr(data-ident ident)',
-				'attr(data-color color)',
-				'attr(data-number number)',
-				'attr(data-percentage percentage)',
-				'attr(data-length length)',
-				'attr(data-angle angle)',
-				'attr(data-time time)',
-				'attr(data-frequency frequency)',
-				'attr(data-flex flex)',
+			tests: [
+				'attr(data-value)',
+				'attr(data-value, "Hello!")',
+				'attr(data-value type(*))',
+				'attr(data-value type(<angle>))',
+				'attr(data-value type(<color>))',
+				'attr(data-value type(<custom-ident>))',
+				'attr(data-value type(<image>))',
+				'attr(data-value type(<integer>))',
+				'attr(data-value type(<length>))',
+				'attr(data-value type(<length-percentage>))',
+				'attr(data-value type(<number>))',
```

Comment:
This case is not present (see in the draft): 
`'attr(data-value type(<number px>, auto))',`

Reply:
The example in the spec. is just a leftover from a previously suggested syntax. `<number px>` is now invalid. See [Tab Atkins' comment](https://github.com/w3c/csswg-drafts/issues/11034#issuecomment-2474389480) and the [related resolution](https://github.com/w3c/csswg-drafts/issues/11035#issuecomment-2474348198).

Therefore, I've [changed the example in the spec. now](https://github.com/w3c/csswg-drafts/commit/f170bb8cdac94fe7dc0d67b84b2c8be6e9891640).

Reply:
Ok, i didn't seen this issue. 😁